.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/plog"]
+	path = third_party/plog
+	url = https://github.com/SergiusTheBest/plog.git
```

Comment:
I cant comment on the submodule itself, but why master branch and not latest release?

Reply:
Good point, will switch to latest available stable tag.